tests/Moq.Tests/CustomTypeMatchersFixture.cs

```diff
@@ -160,6 +162,46 @@ public void It_Is_works_with_custom_matcher()
 			Assert.Equal(2, invocationCount);
 		}
 
+		[Fact]
+		public void It_Is_works_with_custom_comparer()
+		{
+			var acceptableArg = "FOO";
+
+			var invocationCount = 0;
+			var mock = new Mock<IX>();
+			mock.Setup(m => m.Method(It.Is<string>(acceptableArg, StringComparer.OrdinalIgnoreCase)))
+				.Callback((object arg) => invocationCount++);
+
+			mock.Object.Method("foo");
+			mock.Object.Method("bar");
+			Assert.Equal(1, invocationCount);
+
+			mock.Object.Method("FOO");
+			mock.Object.Method("foo");
+			mock.Object.Method((string)null);
+			Assert.Equal(3, invocationCount);
+		}
+
+		[Fact]
+		public void It_Is_object_works_with_custom_comparer()
+		{
+			var acceptableArg = "FOO";
+
+			var invocationCount = 0;
+			var mock = new Mock<IX>();
+			mock.Setup(m => m.Method(It.Is<object>(acceptableArg, new ObjectStringOrdinalIgnoreCaseComparer())))
+				.Callback((object arg) => invocationCount++);
```

Comment:
All good, but I'm not sure I fully understand the rationale behind this test: I'm assuming it's there to ensure that (implicit) type casts won't somehow interfere with the argument matching?

Reply:
Yes kinda. Should I dump this whole test?

Reply:
Yup, that was my intention.

Reply:
Please resolve if you're fine with it.